imports/plugins/core/orders/client/components/lineItems.js

```diff
@@ -61,11 +61,11 @@ class LineItems extends Component {
 
     if (displayMedia(uniqueItem)) {
       return (
-        <img src={displayMedia(uniqueItem).url()}/>
+        <img src={displayMedia(uniqueItem).url()} alt="" />
```

Comment:
Should we put something int he alt tag, instead of leaving it blank?

Reply:
No, as the image is for decoration. https://www.w3.org/WAI/tutorials/images/decorative/ namely these two bullet points:

- Illustrative of adjacent text but not contributing information (“eye-candy”);
- Identified and described by surrounding text.